tests/test_intake_xarray.py

```diff
@@ -218,3 +218,36 @@ def test_read_pattern_path_as_pattern_as_str_with_list_of_urlpaths():
     for color, values in rgb.items():
         for i, v in enumerate(values):
             assert (da.sel(color=color).sel(band=i+1).values == v).all()
+
+
+def test_read_image():
+    pytest.importorskip('skimage')
+    im = intake.open_xarray_image(os.path.join(here, 'data', 'little_red.tif'))
```

Comment:
Should we also test with a 2D, greyscale image?

Reply:
sure might be good. I don't want to be in the business of testing imread but we should be testing our dim mangling.